tasty-query/shared/src/test/scala/tastyquery/ReadTreeSuite.scala

```diff
@@ -1083,6 +1083,55 @@ class ReadTreeSuite extends RestrictedUnpicklingSuite {
     assert(containsSubtree(classWithParams)(clue(tree)))
   }
 
+  testUnpickle("super-accessors", "simple_trees.SuperDoubleDiamond$") { tree =>
+    val ABTemplate = findTree(tree) { case ClassDef(TypeName(SimpleName("AB")), template, _) =>
+      template
+    }
+    val ABCheck: StructureCheck = {
+      case Template(
+            _,
+            _,
+            _,
+            List(
+              DefDef(
+                PrefixedName(
+                  NameTags.SUPERACCESSOR,
+                  ExpandedName(
+                    NameTags.EXPANDED,
+                    ExpandedName(NameTags.EXPANDPREFIX, _, SimpleName("AB")),
+                    SimpleName("foo")
+                  )
+                ),
+                _,
+                _,
+                None,
+                _
+              ),
+              DefDef(
+                SimpleName("foo"),
+                _,
+                _,
+                Some(
+                  Select(
+                    This(TypeIdent(TypeName(SimpleName("AB")))),
+                    PrefixedName(
+                      NameTags.SUPERACCESSOR,
+                      ExpandedName(
+                        NameTags.EXPANDED,
+                        ExpandedName(NameTags.EXPANDPREFIX, _, SimpleName("AB")),
+                        SimpleName("foo")
+                      )
+                    )
```

Comment:
Do you also want to check that the other type of `PrefixedName` appears in the trees? The one corresponding to `super[Mixin].foo` of the form `PrefixedName(_, ExpandedName(EXPANDPREFIX, "foo", "Mixin"))`.